data_structures/Queue_implementation.cpp

```diff
@@ -0,0 +1,54 @@
+// CPP program to implement Queue using
+// two stacks with costly deQueue()
+#include <bits/stdc++.h>
+using namespace std;
```

Comment:
These are discouraged. Please use each header/library you need and strictly use `std::`.

Reply:
ok